setup.py

```diff
@@ -8,9 +8,10 @@
     'pytest-cov==2.4.0',
     'coveralls[yaml]==1.6.0',
     'pytest-xdist==1.18.1',
-    'py-evm==0.2.0a39',
-    'eth-tester==0.1.0b37',
-    'web3==5.0.0a6',
+    'py-evm==0.2.0a42',
+    'eth-tester==0.1.0b39',
+    'eth-abi==2.0.0b9',
+    'web3==5.0.0b2',
```

Comment:
Could this dependency simply be `web3[tester]>=5.0.0b1,<5.1.0`, removing `eth-tester` and `py-evm` above? It will autodownload the correct versions of those two packages required to function, so we don't have pinning problems.

Reply:
Having exact control, allows me to pin versions better - i.e. if I want eth-abi with an existing version (because a eth-tester release wasn't cut yet) that's why it ended up being setup like this.